docs/create-gh-validate-interceptor.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: create-github-validate-interceptor
+spec:
+  inputs:
+    params:
+      - name: Name
+        description: "The name of the interceptor Service."
+      - name: GitHubSecretName
+        description: "The Secret name for GitHub webhook secret. This is always mounted and must exist"
+      - name: GitHubSecretStringKey
+        description: "Key name within the Secret for the Github webhook secret."
+      - name: ContainerImage
```

Comment:
I'm following these docs implementing a validator for our move to use Triggers with our Tekton Webhooks Extension, it's not clear what this container image might do exactly or if this should be replaced to use, say, my own Docker registry if I go on to ko apply the provided Task

Should the container image, say, run some script perhaps that would return a non-zero exit code if some condition fails? 

What would say script look like (I see https://github.com/tektoncd/triggers/blob/6e81d0f3dea0497f70e32cd5dc30285703d451e7/examples/event-interceptors/github-validate.yaml, but I'm used to validation being done through the Task itself like with https://github.com/tektoncd/experimental/pull/258/files#diff-16e2249a36211431f525dd0f47c1703c)

I know this is a WIP so perhaps that's coming or elsewhere!

The use cases we're looking at specifically include:
- Checking valid X-GitHub signature (checking secretToken used in the validation service with what's set on the webhook)
- Checking a repository URL matches an input parameter (so templates/bindings only get used to trigger PipelineRuns/PipelineResource generation for a given repository, not everything we have webhooks for)
- Eventually checking the type of event coming in (e.g. a push or pull request being created, not just anything)

Reply:
So, I tried adding some docs in https://github.com/tektoncd/triggers/blob/6e81d0f3dea0497f70e32cd5dc30285703d451e7/examples/event-interceptors/README.md

but yeah having the task in docs/ and the actual container code in examples/ looks confusing. Open to ideas here. The other tasks are in docs/ so I was following the pattern. Maybe we should have a folder for these tasks somewhere?